### ISSUE TITLE
Unable to import NatWest data

### ISSUE THREAD

**Issue by Ian53OBE** [moneymanagerex/moneymanagerex#6321]
I run MMEX 1.6.4 which is reported as the most recent version. A Mac user with Monterey 12.7.1 installed 

Last week I couldn't import Natwest bank data from a CSV file. It seemed the date format might have changed in the CSV file so after changing the selection on import I got most date to import although only just noticed it hasn't captured payee data. 
Just tried to import new data and nothing will import and I get an error message before running import that there is an error and only description is Vector, see attached 

![Screenshot 2023-11-25 at 12 03 28](https://github.com/moneymanagerex/moneymanagerex/assets/139635837/057fb040-92ba-4b0d-836d-a6e15d80c08d)

 and MMEX version:
 - [ x] 1.6.4
 - [ 1.7.0
 - [ ] 1.7.1
 - [ ] 1.7.2
 - [ ] Other (please specify)

Note: bug reporters are expected to have verified the bug still exists
either in the last stable version of MMEX or on updated development code
(master branch).

Operating System:
 - [x] Windows
 - [ ] Mac OSX
 - [ ] Linux 

# Description of the bug

Please describe your issue with details.
Add screenshot or other files if needed.

# Reproduction

Is the bug reproducible? 
 - [x] Always 
 - [ ] Randomly 
 - [ ] Happened only once

Reproduction steps:

Expected result:

Actual result:

# Additional information

If you have a backtrace for a crash or a warning, paste it here.

**Comment by whalley** [comment]
What are your import settings and can you share the import file please.

**Comment by Ian53OBE** [comment]
Hi Mark,

Very kind of you to respond so quickly.
Bizarrely, almost embarrassingly, when I opened the App to get the information  you asked for the payee details of past transactions that had disappeared reappeared and when I attempted to import the file of transactions I down loaded yesterday it did so without problem. So as now now no problem but I don't understand why. 

I must make a donation to you guys for service even if at the moment I dont seem to need your help.

Regards

Ian
